lib/hanami/assets/configuration.rb

```diff
@@ -460,7 +508,7 @@ def reset!
       #
       # @since 0.1.0
       def load!
-        if digest && manifest_path.exist?
+        if (digest || subresource_integrity) && manifest_path.exist?
           @digest_manifest = Config::DigestManifest.new(
             JSON.load(manifest_path.read),
```

Comment:
I think we should rename some stuff here? since I added the concept of 'fingerprinting' in the `Bundler` namespace.